.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,4 +6,13 @@
 ### What issues does this PR fix or reference?
 
 
-### Specify the version of the product this PR applies to. 
+### Specify the version of the product this PR applies to.
+
+
+### PR Checklist
+
+As the author of this Pull Request I made sure that:
+
+- [ ] Changed articles references are updated where they are used:
+    - [ ] Dashboard [branding.constant.ts](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.constant.ts) + [product.json](https://github.com/eclipse/che-dashboard/blob/master/src/assets/branding/product.json)
+    - [ ] Chectl
```

Comment:
@tolusha Does chectl have one/a couple files which collects all documentation link chectl references to?

Reply:
https://github.com/che-incubator/chectl/blob/master/src/constants.ts

Reply:
thx. I've added that into checklist